aiocache/decorators.py

```diff
@@ -38,8 +38,11 @@ class cached:
     :param key: str value to set as key for the function return. Takes precedence over
         key_builder param. If key and key_builder are not passed, it will use module_name
         + function_name + args + kwargs
+    :param namespace: string to use as default prefix for the key used in all operations of
+        the backend. Default is None
     :param key_builder: Callable that allows to build the function dynamically. It receives
         the function plus same args and kwargs passed to the function.
+        This behavior is necessarily different than ``BaseCache.build_key()``
```

Comment:
I'm looking at the code, and not really seeing the point of this `key_builder`. Maybe we should just remove it and use the cache methods (later)...

Reply:
This `key_builder` in the decorators is necessary to make a cache key from the call signature of the function/coroutine that is being decorated.  It's the only place where the "raw" key gets generated. It accepts the callable object and the `args` + `kwargs` from that callable.

Whereas the `key_builder` supplied to `BaseCache` (which becomes `BaseCache.build_key`) accepts only two parameters: `key` and `namespace`, and it concatenates them by default. The "raw" key must already exist.

Perhaps these two functionalities of theses parameters should have separate names, rather than both being designated as `key_builder`?

Reply:
OK, let's leave it alone for now then. Can probably do with some better documentation or something later though.